definitions/features/pulpcore_database.rb

```diff
@@ -22,6 +23,12 @@ def services
     ]
   end
 
+  def execute_with_db_shell(command_input)
+    cmd = "echo '#{command_input}' | PULP_SETTINGS=#{PULPCORE_DB_CONFIG}"
```

Comment:
What if you replace the single quotes in the echo part with double quotes?

Reply:
Maybe the helper should wrap [open3](https://docs.ruby-lang.org/en/2.0.0/Open3.html) (I'm linking to Ruby 2.0.0 since that's what we need to support) and use `capture2` or `capture3`. That allows you to pass in the SQL via stdin directly rather than via `echo`.

Another option
```suggestion
    cmd = "cat <<SQL | PULP_SETTINGS=#{PULPCORE_DB_CONFIG} PULPCORE_DB_SHELL_COMMAND\n"
    cmd += command_input
    cmd += "\nSQL"
```

Reply:
@jturel the `echo` using single quotes was causing the problem, switching to a double quote works.

@ekohl at some point, should the existing helper functions be looked at for improvements as you suggested? I feel like a big refactor of the `execute` functions might be worth a separate PR.

Reply:
@jturel I still think using `cat` over `echo` is better because it avoids quoting altogether. It should look something like this in pure shell (without the added Ruby on top of it):
```bash
cat <<SQL | PULP_SETTINGS=/etc/pulp/settings.py pulpcore-manager dbshell
SELECT 1
SQL
```

Reply:
@ekohl Let's try the `cat` approach.

Reply:
@ekohl the `cat` strategy works pretty well.

Reply:
Glad to hear that.